source/sync/data-access-patterns/permissions.txt

```diff
@@ -345,6 +345,74 @@ objects regardless of their type. You use :ref:`expressions
 <expressions>` to determine when a role applies and to define dynamic
 field-level permissions for a specific role.
 
+.. _flexible-sync-collection-level-rules:
+
+Collection-Level Rules
```

Comment:
Do we want to introduce the word "collection" here? In a Realm context we can say "Type-specific". There's no obvious connection between object types and "collection". Would it work if this section got merged with the above section?

Reply:
🤔 

Originally, this section was titled "Custom and Default Roles" - but the reason for bringing it back is responding to user who is confused about whether you can assign different permissions on a per-collection basis. And we do talk about collections on the "[Choose Your Sync Mode](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/recover-collection-based-permissions/sync/data-access-patterns/sync-mode/#flexible-sync)" page in the example. I think using this heading makes it scannable on the page for folks who might specifically be looking for collection-level permissions.

Reply:
Either way, doesn't it leave us with two sections (this and the one above) that basically say the same thing in different words? Still not sure why this was removed exactly, but I know we were trying to avoid mixing and matching Realm and MongoDB terminology.

Reply:
Hmm, ok. I've updated it a bit to combine it with the section above and use more of the type-specific language. I did leave collections in a couple of places because I think you can infer that the collection is a collection of the specified type in context, without needing to know that a collection is a MongoDB term. What do you think of the updated version?

As a general note, though, collection is a term used all over the App Services UI, including when configuring Rules and Device Sync. It's probably a good idea to use terms in the App Services docs that are consistent with the App Services UI, and if that differs from the Realm term and we're discussing a Realm concept, just clearly state somewhere how the two terms are related.